index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Anja Schmidt's Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="mobile_container">
```

Comment:
[OPTIONAL] I want to recommend you to not use this container, instead, use the dev tools from your web browser to resize the window of your web app when you're working on the mobile version. Your page will automatically resize to any viewport and in upcoming lessons, you'll learn how to modify its behavior based on the width of the window.

I hope you find this useful! :slightly_smiling_face: 

![image](https://user-images.githubusercontent.com/79114233/138181832-972245d6-ce6f-4e13-9eed-8594ddc5a28a.png)

This is a preview of how you can use the dev tools to resize the viewport.